README.md

```diff
@@ -39,6 +39,13 @@ asciidoctor-pdf \
     -a pdf-style=owncloud
 ```
 
+---
+
+**Note:** the custom ownCloud PDF theme references a custom font, *Times New Roman*, in place of the packaged Times Roman font, as Times Roman doesn't support a wide enough character set to render the manuals correctly.
+
+If you have this font available on your system, you need to copy it (all four variants: _normal_, _italic_, _bold_, and _bold-italic_), to the `fonts` directory. If you don't, then you'll have to copy the closest matching font family to Times New Roman that you do have, and then update `resources/themes/owncloud-theme.yml`.
+
```

Comment:
where did you got the custom font from?
that might be a valuable note for others... (...If you don't, then you'll have to...)

Reply:
This would be at least a good hint.
eg, on macOS, you can find the the font <whatever you have taken and is the closest match> in <....>, please look on other OS accordingly (make that sentence looking nice, but you understand the intention).
How is the naming of the font family files, do they have a particular extension (for eas of search),
are these fonts maybe available to be downloaded on the internet?

Reply:
Well said. I'll make the required changes.

Reply:
277cc1b updates the previous commit to:

- list the default font directories (as best I understand them)
- link to further information per platform
- link to font directories